locust/run-load-test

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import sys, subprocess, os
+
+def usage():
+    usage = """
+This script only adds defaults if there are no options
+Usage: run-load-test [options]
+
+Defaults:
+  -H [--host]       http://localhost:8080
+  -u [--users]      1
+  -s [--spawn_rate] 1
+  -t [--run_time]   1m
+  -f [--locustfile] locust/load-test/load-test.py
+  --html            report.html
+"""
+    print(usage)
+
+def add_default_args(program_args):
+    defaults = [['--headless'], ['-H', 'http://localhost:8080'],
+           ["-u", '1'], ["-r", '1'], ["-t", '1m'], ['-f', locust_filepath], ['--html', 'report.html']]
+    additional_args = []
+    for default_arg in defaults:
+        found = False
+        for arg in program_args:
+            if arg == default_arg[0]:
+                found = True
+        if not found:
+            additional_args += default_arg
+
+    return additional_args + program_args
+
+subprocess.run([sys.executable, "-m", "pip", "install", "locust", "pyyaml", "-U"])
```

Comment:
is this still necessary, if in the docs, you suggest installing locust before running the load test?

Reply:
I've updated the docs. We need to update the locust gem because we use the `--html` option that was added in 1.4.2 which is quite recent.